### PR TITLE
Support use of super inside instance setter.

### PR DESCRIPTION
**Pull request by kruppel**
This change allows for subclasses to use super in instance setters.

##### Example

```javascript
class ClassA {
  get foo() {
    return this.x;
  }

  set foo(x) {
    this.x = x;
  }
}

class ClassB extends ClassA {
  get foo() {
    return super.foo;
  }

  set foo(x) {
    super.foo = x + 'bar';
  }
}
```

@eventualbuddha Tagging you since worked on this rather recently.

### CODE REVIEW

File: lib/6to5/transformation/helpers/replace-supers.js [hunk 1]
```diff
@@ -25,6 +25,36 @@ function ReplaceSupers(methodNode, className, superName, isLoose, file) {
...
+    this.file.addHelper("set"),
+    [
+      t.callExpression(
+        t.memberExpression(t.identifier("Object"), t.identifier("getPrototypeOf")),
+        [
+          isStatic ? this.className : t.memberExpression(this.className, t.identifier("prototype"))
+        ]
+      ),
+      isComputed ? property : t.literal(property.name),
+      t.identifier(value.name),
```
Comment (kruppel) [inline lib/6to5/transformation/helpers/replace-supers.js]: This works fine for simple expressions with Identifiers. If the right hand side of the assignment expression is something more complex though, things go awry.

Comment (sebmck) [inline lib/6to5/transformation/helpers/replace-supers.js]: Is there a reason you can't just insert the right side verbatim?

Comment (kruppel) [inline lib/6to5/transformation/helpers/replace-supers.js]: Ya, that works perfectly actually.

File: test/fixtures/esnext/es6-classes/getter-setter-super.js [hunk 2]
```diff
@@ -12,6 +20,18 @@ class Cat extends Animal {
   get sound() {
     return super.sound + ' MEOW!';
   }
+
+  get name() {
+    return super.name;
+  }
```
Comment (kruppel) [inline test/fixtures/esnext/es6-classes/getter-setter-super.js]: Should this be necessary? Without a name getter declared, `cat.name` always returns `undefined`. Everything works fine if a name property is defined in the immediate parent.

**Comment by kruppel** [comment]
@sebmck Are there any other changes required?

**Comment by sebmck** [comment]
@kruppel Sorry I haven't been able to get around to this! I've been pretty busy getting the 3.0.0 release ready. I'll put some time aside tonight to merge this in and release a minor if necessary. Thanks!

**Comment by kruppel** [comment]
No worries! Do your thing. Appreciate all the hard work. Just wanted to know if I should make any changes.

**Comment by sebmck** [comment]
Awesome, this is great. Thanks! Sorry I didn't get around to merging this sooner.

**Comment by sebmck** [comment]
Released as of 3.2.0, thanks again @kruppel!

**Pull request merged**